### PR TITLE
eval wer with gpu

### PR DESCRIPTION
**Pull request by nobel861017**
In eval.py line 74, pass ```conf["use_gpu"]``` to ```WERTracker```.
In metrics.py line 231 add ```self.device = 'cuda' if use_gpu else 'cpu'``` and pass ```device=self.device``` to ```Speech2Text```.

### CODE REVIEW

File: asteroid/metrics.py [hunk 1]
```diff
@@ -342,6 +344,7 @@ def hsdi(truth, hypothesis, transformation):
         return {k: v for k, v in out if k in keep}
 
     def predict_hypothesis(self, wav):
+        wav = torch.from_numpy(wav).cuda()
```
Comment (mpariente) [inline asteroid/metrics.py:347]: That does not seem right? Isn't is `.to(self.device)`?

File: asteroid/metrics.py [hunk 2]
```diff
@@ -220,15 +221,16 @@ class WERTracker:
             See librimix/ConvTasNet recipe.
     """
 
-    def __init__(self, model_name, trans_df):
+    def __init__(self, model_name, trans_df, use_gpu):
```
Comment (mpariente) [inline asteroid/metrics.py:224]: Let's make it a keyword argument.

File: egs/librimix/ConvTasNet/eval.py [hunk 3]
```diff
@@ -71,7 +71,7 @@ def main(conf):
     compute_metrics = update_compute_metrics(conf["compute_wer"], COMPUTE_METRICS)
     anno_df = pd.read_csv(Path(conf["test_dir"]).parent.parent.parent / "test_annotations.csv")
     wer_tracker = (
-        MockWERTracker() if not conf["compute_wer"] else WERTracker(ASR_MODEL_PATH, anno_df)
+        MockWERTracker() if not conf["compute_wer"] else WERTracker(ASR_MODEL_PATH, anno_df, conf["use_gpu"])
```
Comment (mpariente) [inline egs/librimix/ConvTasNet/eval.py:74]: Keyword argument here too.

**Review by mpariente — COMMENTED** [review]
Thanks for the PR ! 
Minor things to be changed and we're good to go.

File: asteroid/metrics.py (hunk 1 above)
Comment (nobel861017) [reply asteroid/metrics.py:347]: Oh yeah. My bad.

File: egs/librimix/ConvTasNet/eval.py (hunk 3 above)
Comment (nobel861017) [reply egs/librimix/ConvTasNet/eval.py]: fixed

File: asteroid/metrics.py (hunk 2 above)
Comment (nobel861017) [reply asteroid/metrics.py]: fixed

File: asteroid/metrics.py (hunk 1 above)
Comment (nobel861017) [reply asteroid/metrics.py]: fixed

File: asteroid/metrics.py [hunk 4]
```diff
@@ -220,15 +221,16 @@ class WERTracker:
             See librimix/ConvTasNet recipe.
     """
 
-    def __init__(self, model_name, trans_df):
+    def __init__(self, model_name, trans_df, use_gpu=1):
```
Comment (mpariente) [inline asteroid/metrics.py:224]: use_gpu should be a boolean, so default to True if you want. Plus add arg to docstring.

Comment (nobel861017) [reply asteroid/metrics.py]: fixed

**Comment by mpariente** [comment]
Looks good to me. Can you lint the PR please?

**Pull request merged**